docs/docs/policies/actions.md

```diff
@@ -1,13 +1,47 @@
 # Actions
 
-Actions are taken when there is an event match. Currently the only action supported is `log`. 
+## DefaultAction
+Every policy must have a `defaultAction`, actions are taken when there is match on some rule declared at the policy. 
+The following actions are currently supported:
+
+- `log` - output events in json format. The default path to file is stdout.
+- `forward:http://url/fluent` - send events in json format using the Forward protocol to a Fluent receiver
+- `webhook:http://url/webhook` - send events in json format to the webhook url
+
+## Examples
+
+### Log action
 
 ```yaml
-name: overview policy
-description: sample overview policy
+name: log_sample_policy
+description: log sample policy
 scope:
   - global
 defaultAction: log
 rules:
   - event: dropped_executable
-```
\ No newline at end of file
+```
+
+### Webhook action
+
+```yaml
+name: webhook_sample_policy
+description: webhook_sample_policy
+scope:
+  - global
+defaultAction: webhook:http://localhost:8080
+rules:
+  - event: dropped_executable
+```
+
+### Fluentd action
+
+```yaml
+name: webhook_sample_policy
+description: webhook_sample_policy
+scope:
+  - global
+defaultAction: forward:tcp://localhost:24224
+rules:
+  - event: dropped_executable
+```
```

Comment:
Should these also be `examples/` files?

Reply:
don't think so, in the example we use logs because the user can apply at will and see the result, for webhook/fluentd the user needs to do other steps, so only docs is enough.